storage/rocksdb/rdb_threads.cc

```diff
@@ -68,7 +68,7 @@ int Rdb_thread::create_thread(
 }
 
 
-void Rdb_thread::signal(bool stop_thread)
+void Rdb_thread::signal(const bool &stop_thread)
```

Comment:
It comes from upstream, and it was an intentional change, but I have no idea why. I agree 100%, there is no need to introduce the extra needed dereferencing when by-value is so minimal.